# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/gnome-screenshot/hotfix-meson.patch
# Copyright (C) 2026 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- gnome-screenshot-41.0/data/meson.build.vanilla	2026-09-27 23:46:13.988390323 +0300
+++ gnome-screenshot-41.0/data/meson.build	2026-09-27 23:46:41.551191767 +0300
@@ -1,7 +1,6 @@
 gnome = import('gnome')
 
 desktop_file = i18n.merge_file(
-  'desktop',
   type: 'desktop',
   input: 'org.gnome.Screenshot.desktop.in',
   output: 'org.gnome.Screenshot.desktop',
@@ -23,7 +22,6 @@
 endif
 
 metainfo_file = i18n.merge_file(
-  'appdata',
   input: 'org.gnome.Screenshot.metainfo.xml.in',
   output: 'org.gnome.Screenshot.metainfo.xml',
   po_dir: join_paths(meson.current_source_dir(), '../po'),
